src/components/Events.vue

```diff
@@ -46,47 +46,41 @@ export default {
     this.events = [
       {
         id: 0,
-        date: getDateParts(new Date('2021-08-24')),
-        title: 'Workshop: Practicing Computer Skills with OverTheWire',
-        description: 'OverTheWire (https://overthewire.org/wargames/) is a platform that hosts wargames to learn and practice security and computer concepts in the form of games. The platform has several categories with different difficulties. \n\nThis event is for everyone. Something for everyone due to the categories and levels. This is especially good for first-year students to learn about common computer concepts and get comfortable with the Linux environment (which is really important when doing programming and tons of other computer stuff). You literally don\'t need to know _anything_ about CTF or security to join in!'
+        date: getDateParts(new Date('2022-01-26')),
+        title: 'Workshop: Machine setup - UIO and online - 17.00-20.00',
+        description: 'Having a proper environment on your machine is essential to do efficient programming and solving CTF challenges. We host this workshop to provide you with advices on software to use, as well as helping you on your way to install them.
+        In our previous workshops we've helped a lot of you to setup Windows Subsystem for Linux (WSL) or install virtual machines to run a Linux distro. Don't know what WSL or Linux is? Don't worry! We will introduce our workshop with the required information.
+        Target audience: Everyone who would like a few advices of software to check out, or want some help with setting up software for your environment. Or just hang out with us <3'
```

Comment:
Jeg ser at teksten inneholder en enkeltfnutt `'` som også er det samme som description-verdien bruker for å inkapsulere teksten. Det betyr at den tror at teksten slutter med `In our previous workshop we` og ikke skjønner hva som foregår etterpå.

Kanskje det fungerer med å escape den sånn som dette: `\'`, eller at vi kan bruke `"` rundt verdien i stedet. Ikke helt sikker.